lib/simulator-xcode-6.js

```diff
@@ -290,6 +291,10 @@ class SimulatorXcode6 extends EventEmitter {
     if (!this.connectHardwareKeyboard) {
       args.push('-ConnectHardwareKeyboard', '0');
     }
+
+    // Do the 'Touch ID Enroll' key bindings before the Simulator starts
+    await setTouchEnrollKey();
```

Comment:
Yep, you got it. The reason for the cap is to avoid this system tomfoolery for those who don't need the functionality, just to be safe.